samples/ingress/path-based-ingress/logging/user-values-elasticsearch-open.yaml

```diff
@@ -28,6 +28,8 @@ kibana:
         rewrite (?i)/kibana$ / break;
       nginx.ingress.kubernetes.io/rewrite-target: /kibana
     enabled: true
+    # ingressClass is not currently supported by the Open Distro helm chart
+    # ingressClassName: nginx
```

Comment:
Comments on lines 31 and 32 are redundant, duplicating the existing comments on lines 21 and 22.

Reply:
fixed